spec/index.html

```diff
@@ -172,61 +172,67 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-triple-terms">
+    <h3>Triple Terms</h3>
 
-    <p class="issue" data-number="81">
-      As the WG has moved away from the notion of quoted triples, this section needs to be changed such that it does not mention quoted triples anymore but, instead, describes whatever new concept the WG settles on (e.g., <a>triple terms</a>).
-    </p>
-
-    <p>A <dfn>quoted triple</dfn> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>triple term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF triple</a>, which can be used as the <a>object</a>
       of another triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>A <a>triple term</a> is not necessarily asserted, allowing
       statements to be made about other statements that may not be
       asserted within an <a>RDF graph</a>.
       This allows statements to be made about relationships that may be contradictory.
-      For a <a>quoted triple</a> to be asserted,
+      For a <a>triple term</a> to be asserted,
       it must also appear in a graph as an <a>asserted triple</a>.</p>
 
-    <p>The following diagram represents a statement with an unasserted <a>quoted triple</a>
-      as the subject.</p>
-
-    <figure id="fig-quoted-triple">
-      <a href="quoted-triple.svg">
-        <!-- Source for this file is at https://docs.google.com/drawings/d/1I_QxbUgnQXumXzb8c0WNJHIQ-mtRs2S80dDG6i9aOD8 -->
-        <img src="quoted-triple.svg"
-             alt="An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject"
+    <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>.
+      The <a>subject</a> of that <a>triple</a> is called a <dfn>reifier</dfn>.
```

Comment:
Given that this section is non-normative, I wonder whether it makes sense to put a definition here (`<dfn>reifier</dfn>`) ?

Reply:
As a reifier is really just a concept, and not a component of the abstract syntax itself, it seems to me that a non-normative definition is fine. I think we also need a concept for "reified triple term", or whatever we come up with. In Turtle, both "reifier" and "reified triple term" will be normatively defined as part of the grammar, and it may be elsewhere, so having the source definition in RDF Concepts makes sense.

But, I'm open to suggestions to handle this better.

Reply:
Makes sense.